[BUG] GPT 5.5 model fails with StreamingException due to temperature validation

## 📚 Before Submitting

**Please check our [documentation](https://genie.devoxx.com/docs/getting-started/introduction) first!**

You may also find helpful information in our [Features Overview](https://genie.devoxx.com/docs/features/overview) or search [existing issues](https://github.com/devoxx/DevoxxGenieIDEAPlugin/issues).

## Bug Description

When using the OpenAI model **GPT 5.5** the plugin throws a `StreamingException` because the API rejects a temperature value of `0`. 

## Steps to Reproduce

1. Open the Devoxx Genie chat panel.
2. Select **OpenAI** as the provider.
3. Select the model **GPT 5.5**.
5. Submit a prompt.

## Expected Behavior

The message should be returned 

## Actual Behavior

The request fails with the following error:
```
StreamingException - Error during streaming response Caused by: InvalidRequestException - { "error": { "message": "Unsupported value: 'temperature' does not support 0 with this model. Only the default (1) value is supported.", "type": "invalid_request_error", "param": "temperature", "code": "unsupported_value" } } Please check the IDE log for more details.
```

## Screenshots/Logs

<img width="378" height="200" alt="Image" src="https://github.com/user-attachments/assets/f1825971-9709-41bb-8586-ee95a4c98afc" />

```json
{
  "error": {
    "message": "Unsupported value: 'temperature' does not support 0 with this model. Only the default (1) value is supported.",
    "type": "invalid_request_error",
    "param": "temperature",
    "code": "unsupported_value"
  }
}
```

## Environment

- **IntelliJ IDEA Version**: [e.g. 2026.1]
- **Plugin Version**: [e.g. 1.7.0]
- **Operating System**: [e.g. Ubuntu 24.04.4 LTS]
- **LLM Provider**: OpenAI
- **Model**: GPT 5.5

--- a/.gitignore
+++ b/.gitignore
@@ -25,3 +25,4 @@ docusaurus/static/img/git-diff-viewer.png
 .project
 .settings/
 *.class
+out/*

--- a/src/main/java/com/devoxx/genie/chatmodel/cloud/openai/OpenAIChatModelFactory.java
+++ b/src/main/java/com/devoxx/genie/chatmodel/cloud/openai/OpenAIChatModelFactory.java
@@ -48,13 +48,20 @@ public class OpenAIChatModelFactory implements ChatModelFactory {
 
     private ChatRequestParameters createChatContextParameters(@NotNull CustomChatModel customChatModel) {
         String modelName = customChatModel.getModelName().toLowerCase();
-        boolean isReasoningModel = modelName.startsWith("o1") || modelName.startsWith("o3");
-        boolean isGpt5 = modelName.startsWith("gpt-5");
 
-        if (isReasoningModel) {
+        if (modelName.startsWith("o1") || modelName.startsWith("o3")) {
             // o1 and o3 models do not support temperature and topP
             return ChatRequestParameters.builder().build();
-        } else if (isGpt5) {
+        } else if (modelName.startsWith("gpt-5.5")) {
+            double temperature = customChatModel.getTemperature();
+            // GPT-5.5 requires temperature to be 1
+            if (temperature==0) {
+                temperature = 1d;
+            }
+            return ChatRequestParameters.builder()
+                    .temperature(temperature)
+                    .build();
+        } else if (modelName.startsWith("gpt-5")) {
             // GPT-5 supports temperature but not topP
             return ChatRequestParameters.builder()
                     .temperature(customChatModel.getTemperature())

